pom.xml

```diff
@@ -40,4 +41,57 @@
 			</snapshots>
 		</repository>
 	</repositories>
+	
+	<profiles>
+		<profile>
+			<id>quality</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
```

Comment:
good catch. Copy-pasted it from jbostools-openshift. Need to fix it in both here and there now. Left after experimenting with sonar. Then forgot to remove